docs/cody/troubleshooting.mdx

```diff
@@ -137,3 +137,11 @@ JetBrains logs can be accessed via the **Output** panel. To access logs, you mus
 - Click Apply
 - To access the logs, go to Help - Show Log in Finder
 - Open the `idea.log` file
+
+### High CPU Usage with Cody Agent
+
+If you notice the Cody agent reaching 100% CPU utilization, try the following:
+
+1. Disable the Cody plugin.
+1. Re-enable the plugin.
```

Comment:
I don't know if this is a GitHub preview issue, but the second iteration also shows 1.